src/main/java/org/graphqlcrudjava/DatabaseAnalyzer.java

```diff
@@ -83,22 +82,23 @@ public Entity initializeEntity(String table) throws SQLException {
                 entity.addAttribute(attribute);
             }
         }
-
         return entity;
     }
 
     public Attribute initializeAttribute(ResultSet results, List<String> primaryKeys) throws SQLException {
+        List<GraphQLArgument> graphQLArguments = Collections.emptyList();
```

Comment:
```suggestion
        List<GraphQLArgument> graphQLArguments = new ArrayList<>();
```

We might want to use this instead if we plan to modify the list later on.

Reply:
Yes we might need it later on.